### ISSUE TITLE
Explicitly including a reference to "Microsoft.NetCore.App" results in lots of errors opening RESX, project faults and analyzer errors

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn-project-system#1555]
1. Open https://github.com/Ponant/ResBugV2

Ran into various issues:

Open Resources\Controllers.HomeController.fr.resx:

![image](https://cloud.githubusercontent.com/assets/1103906/22921891/91a47bb0-f2ef-11e6-8939-62c95e82a59e.png)

```
Severity	Code	Description	Project	File	Line	Suppression State
Message		Object reference not set to an instance of an object.			0	
Message		Object reference not set to an instance of an object.			0	
Message		Object reference not set to an instance of an object.			0	
Message		Object reference not set to an instance of an object.			0	
```
Project fault various times that I opened the project:

```
=====================
02/14/17 7:40:37 PM
Crippling
System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReference.get_Name()
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.EnsurePopulated(Boolean fPopulateProject)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.SearchNormalEntries(String fullName, Boolean correctThread)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.System.ComponentModel.Design.IDesignTimeAssemblyLoader.GetTargetAssemblyPath(AssemblyName runtimeOrTargetAssemblyName, String suggestedAssemblyPath, FrameworkName targetFramework)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.ResolveAssembly(AssemblyName assemblyName)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetSystemAssembly()
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse..ctor(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetUniverse(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkProvider..ctor(IVsDesignTimeAssemblyResolution assemblyResolution, IDesignTimeAssemblyLoader assemblyLoader, TypeDescriptionProvider parentProvider, IVsSmartOpenScope openScope)
   at Microsoft.VisualStudio.ProjectUtilities.GetTargetFrameworkProvider(IServiceProvider provider, IDesignTimeAssemblyLoader assemblyLoader, IVsHierarchy hierarchy, Object instance)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.get_TypeProvider()
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.VSLangProj._dispReferencesEvents.ReferenceAdded(Reference r)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReferences.OnReferenceAdded(Reference reference)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ReferencesHostBridge.ApplyAsync(ReferencesDelta value)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_1.<<InitializeCoreAsync>b__3>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask.<JoinAsync>d__76.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask`1.<JoinAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_0.<<InitializeCoreAsync>b__2>d.MoveNext()
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReference.get_Name()
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.EnsurePopulated(Boolean fPopulateProject)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.SearchNormalEntries(String fullName, Boolean correctThread)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.System.ComponentModel.Design.IDesignTimeAssemblyLoader.GetTargetAssemblyPath(AssemblyName runtimeOrTargetAssemblyName, String suggestedAssemblyPath, FrameworkName targetFramework)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.ResolveAssembly(AssemblyName assemblyName)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetSystemAssembly()
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse..ctor(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetUniverse(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkProvider..ctor(IVsDesignTimeAssemblyResolution assemblyResolution, IDesignTimeAssemblyLoader assemblyLoader, TypeDescriptionProvider parentProvider, IVsSmartOpenScope openScope)
   at Microsoft.VisualStudio.ProjectUtilities.GetTargetFrameworkProvider(IServiceProvider provider, IDesignTimeAssemblyLoader assemblyLoader, IVsHierarchy hierarchy, Object instance)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.get_TypeProvider()
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.VSLangProj._dispReferencesEvents.ReferenceAdded(Reference r)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReferences.OnReferenceAdded(Reference reference)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ReferencesHostBridge.ApplyAsync(ReferencesDelta value)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_1.<<InitializeCoreAsync>b__3>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask.<JoinAsync>d__76.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask`1.<JoinAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_0.<<InitializeCoreAsync>b__2>d.MoveNext()<---

===================
```

Also got the following analyzer errors, which I'm sure if they are related or not (I suspect they are because the TypeResolutionService hooks onto AssemblyResolve, and throws in that path):

```
Severity	Code	Description	Project	File	Line	Suppression State
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.VisualBasic.EditorFeatures.dll: Object reference not set to an instance of an object.			1	Active
Warning	CS8034	Unable to load Analyzer assembly C:\Users\davkean\.nuget\packages\microsoft.codeanalysis.analyzers\1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.Analyzers.dll: Object reference not set to an instance of an object.	LocalizationGarage		1	Active
Warning	CS8034	Unable to load Analyzer assembly C:\Users\davkean\.nuget\packages\microsoft.codeanalysis.analyzers\1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.CSharp.Analyzers.dll: Object reference not set to an instance of an object.	LocalizationGarage		1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.EditorFeatures.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.VisualBasic.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICESNEXT\Microsoft.VisualStudio.LanguageServices.Xaml.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\XAMLLANGUAGESERVICE\Microsoft.VisualStudio.DesignTools.XamlLanguageService.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.Features.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\TYPESCRIPT\Microsoft.CodeAnalysis.TypeScript.EditorFeatures.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.CSharp.EditorFeatures.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.CSharp.Features.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.VisualBasic.Features.dll: Object reference not set to an instance of an object.			1	Active
Warning	AD1002	Unable to load Analyzer assembly C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\ENTERPRISE\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\MANAGEDLANGUAGES\VBCSHARP\LANGUAGESERVICES\Microsoft.CodeAnalysis.CSharp.dll: Object reference not set to an instance of an object.			1	Active
```

```

**Comment by davkean** [comment]
Filed: https://devdiv.visualstudio.com/DevDiv/_workitems?id=384364&fullScreen=false&_a=edit to track this.

**Comment by srivatsn** [comment]
Closing since this is tracked by CPS.

**Comment by lifengl** [comment]
Some code doesn't handle references coming from the NuGet JSON file, and need be fixed.

**Comment by davkean** [comment]
This will be fixed in the first update post RTM. If you run into this, the workaround is to remove the explicit package reference to Microsoft.NETCoreApp.

**Comment by YZahringer** [comment]
Similar error, without reference to "Microsoft.NetCore.App" with VS2017 RTM:
```
=====================
08/03/2017 16:41:34
Crippling
System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReference.get_Version()
   at Microsoft.VisualStudio.ProjectSystem.VS.References.DesignTimeAssemblyResolution.GetAllResolvedReferences()
   at Microsoft.VisualStudio.ProjectSystem.VS.References.DesignTimeAssemblyResolution.ResolveReferences(String[] originalNames, AssemblyName[] assemblyName, VsResolvedAssemblyPath[] assemblyPaths)
   at Microsoft.VisualStudio.ProjectSystem.VS.References.DesignTimeAssemblyResolution.ResolveAssemblyPathInTargetFx(String[] prgAssemblySpecs, UInt32 cAssembliesToResolve, VsResolvedAssemblyPath[] prgResolvedAssemblyPaths, UInt32& pcResolvedAssemblyPaths)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.ResolveAssemblyPath(String assemblySpec)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.ResolveAssembly(AssemblyName assemblyName)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetSystemAssembly()
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse..ctor(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetUniverse(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkProvider..ctor(IVsDesignTimeAssemblyResolution assemblyResolution, IDesignTimeAssemblyLoader assemblyLoader, TypeDescriptionProvider parentProvider, IVsSmartOpenScope openScope)
   at Microsoft.VisualStudio.ProjectUtilities.GetTargetFrameworkProvider(IServiceProvider provider, IDesignTimeAssemblyLoader assemblyLoader, IVsHierarchy hierarchy, Object instance)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.get_TypeProvider()
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.ClearReferenceEntries(Boolean normal, Boolean project, Boolean disposing)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.VSLangProj._dispReferencesEvents.ReferenceRemoved(Reference r)
   at VSLangProj._dispReferencesEvents_ReferenceRemovedEventHandler.Invoke(Reference pReference)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReferences.OnReferenceRemoved(Reference reference)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ReferencesHostBridge.ApplyAsync(ReferencesDelta value)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_1.<<InitializeCoreAsync>b__3>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask.<JoinAsync>d__76.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask`1.<JoinAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_0.<<InitializeCoreAsync>b__2>d.MoveNext()
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object.
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReference.get_Version()
   at Microsoft.VisualStudio.ProjectSystem.VS.References.DesignTimeAssemblyResolution.GetAllResolvedReferences()
   at Microsoft.VisualStudio.ProjectSystem.VS.References.DesignTimeAssemblyResolution.ResolveReferences(String[] originalNames, AssemblyName[] assemblyName, VsResolvedAssemblyPath[] assemblyPaths)
   at Microsoft.VisualStudio.ProjectSystem.VS.References.DesignTimeAssemblyResolution.ResolveAssemblyPathInTargetFx(String[] prgAssemblySpecs, UInt32 cAssembliesToResolve, VsResolvedAssemblyPath[] prgResolvedAssemblyPaths, UInt32& pcResolvedAssemblyPaths)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.ResolveAssemblyPath(String assemblySpec)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.ResolveAssembly(AssemblyName assemblyName)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetSystemAssembly()
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse..ctor(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkUniverse.GetUniverse(IDesignTimeAssemblyLoader assemblyLoader, IVsDesignTimeAssemblyResolution projectAssemblyResolution, IVsSmartOpenScope dispenser)
   at Microsoft.VisualStudio.Shell.Design.VsTargetFrameworkProvider..ctor(IVsDesignTimeAssemblyResolution assemblyResolution, IDesignTimeAssemblyLoader assemblyLoader, TypeDescriptionProvider parentProvider, IVsSmartOpenScope openScope)
   at Microsoft.VisualStudio.ProjectUtilities.GetTargetFrameworkProvider(IServiceProvider provider, IDesignTimeAssemblyLoader assemblyLoader, IVsHierarchy hierarchy, Object instance)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.get_TypeProvider()
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.ClearReferenceEntries(Boolean normal, Boolean project, Boolean disposing)
   at Microsoft.VisualStudio.Design.VSTypeResolutionService.VSLangProj._dispReferencesEvents.ReferenceRemoved(Reference r)
   at VSLangProj._dispReferencesEvents_ReferenceRemovedEventHandler.Invoke(Reference pReference)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.Automation.OAVSReferences.OnReferenceRemoved(Reference reference)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ReferencesHostBridge.ApplyAsync(ReferencesDelta value)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_1.<<InitializeCoreAsync>b__3>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask.<JoinAsync>d__76.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask`1.<JoinAsync>d__3.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.Package.ProjectHostBridge`3.<>c__DisplayClass36_0.<<InitializeCoreAsync>b__2>d.MoveNext()<---

===================
```

**Comment by jaredpar** [comment]
> This will be fixed in the first update post RTM. If you run into this, the workaround is to remove the explicit package reference to Microsoft.NETCoreApp.

I'm also hitting this without an apparent reference to Microsoft.NetCoreApp.  

Here are the repo steps, unsure if it's repeatable or not at the moment. 

1. Clone https://github.com/jaredpar/roslyn
1. Sync to branch repo-ps1 
1. Run `msbuild /t:Restore Compilers.sln`
1. Open Compilers.sln in Visual Studio

**Comment by davkean** [comment]
Yes a reference to Microsoft.NETCoreApp was a symptom, but not the cause. This will happen in normal projects under specific circumstances, it's fixed in 15.1
